samples/matter/window_covering/sysbuild/mcuboot/boards/nrf54lm20dk_nrf54lm20a_cpuapp.conf

```diff
@@ -1,26 +1,3 @@
-#
```

Comment:
Why did you remove all things from here and there? We can just add the proposed note regarding KRKNWK-20774 and set `CONFIG_MINIMAL_LIBC=y`, no?